org.testeditor.web.dropwizard/src/main/java/org/testeditor/web/dropwizard/auth/JWTAuthenticator.xtend

```diff
@@ -1,26 +1,35 @@
 package org.testeditor.web.dropwizard.auth
 
+import com.auth0.jwt.JWT
+import com.auth0.jwt.interfaces.DecodedJWT
 import io.dropwizard.auth.AuthenticationException
 import io.dropwizard.auth.Authenticator
 import java.util.Optional
 
 class JWTAuthenticator implements Authenticator<String, User> {
 
-//	@Inject Provider<Key> keyProvider
+	static val HEADER_PREFIX = "Bearer "
 
-	override authenticate(String compactJws) throws AuthenticationException {
-//		try {
-			// TODO this is a dummy impl only...
-//			val claims = Jwts.parser.setSigningKey(keyProvider.get).parseClaimsJws(compactJws)
-//			val user = claims.body.subject
-//			val email = claims.body.get("email") as String
-			val split = compactJws.split(":")
-			val user = split.head
-			val email = split.last
-			return Optional.of(new User(user, email))
-//		} catch (SignatureException e) {
-//			return Optional.empty
-//		}
+	override authenticate(String header) throws AuthenticationException {
```

Comment:
consider calling the parameter authHeader since it is not the full http header that is passed as a String